src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -4254,5 +4257,36 @@ GUI_TEST_CLASS_DEFINITION(test_7806) {
     CHECK_SET_ERR(size == 4, "chrM.fa in SAM dir is changed, size: " + QString::number(size));
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7842) {
+    // 1. Open "GenBank/murine.gb" and "GenBank/sars.gb"
+    // 2. Click right button->Cloning->Construct molecule...
+    // 3. Click "From project..."
+    // 4. Choose both sequences
+    // Expected: only one sequence could be selected, only one "Create Fragment" dialog has appeared, only one fragment added
+
+    GTUtilsDialog::waitForDialog(os, new GTSequenceReadingModeDialogUtils(os));
+    GTFileDialog::openFileList(os, dataDir + "samples/GenBank", { "murine.gb", "sars.gb" });
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    class Scenario : public CustomScenario {
+        void run(HI::GUITestOpStatus& os) {
+            QWidget* dialog = GTWidget::getActiveModalWidget(os);
+
+            GTUtilsDialog::waitForDialog(os, new CreateFragmentDialogFiller(os));
+            GTUtilsDialog::waitForDialog(os, new ProjectTreeItemSelectorDialogFiller(os, { { "sars.gb", { "NC_004718" } }, { "murine.gb", { "NC_001363" } } } ));
+            GTWidget::click(os, GTWidget::findWidget(os, "fromProjectButton"));
+
+            auto fragmentListWidget = GTWidget::findListWidget(os, "fragmentListWidget", dialog);
+            CHECK_SET_ERR(fragmentListWidget->count() == 1, QString("Unexpected fragments size, expected: 1, current: %1").arg(fragmentListWidget->count()));
```

Comment:
You ask to select 3 and result with 1. 
The filler should fail, but not silently return 1, because it can't fulfill the request.

Reply:
Don't get, what you mean and what you want to change

Reply:
You select 2 objects in Project view:
```
GTUtilsDialog::waitForDialog(os, new ProjectTreeItemSelectorDialogFiller(os, { { "sars.gb", { "NC_004718" } }, { "murine.gb", { "NC_001363" } } } ));
```

and say in the description:

```
Only one sequence could be selected
```

Could you confirm that ProjectTreeItemSelectorDialogFiller selects both, right? In this case you need to change the description